assets/script/script.js

```diff
@@ -31,6 +31,15 @@ $(document).ready(function () {
       }
     }
   });
+
+  //Listener on clear button clears the history list
+  $(".shiftRight").on("click", function () {
+    window.localStorage.clear("history");
+    window.sessionStorage.clear();
```

Comment:
I think we should update the html for this to be a link with an id. That way we can listen here for something like...
```
$("#clearLink").on("click", ...
```
.shiftRight is a class and could be used elsewhere.

Reply:
Yes, I agree.